tasks/upload_processor.py

```diff
@@ -80,6 +80,7 @@ def run_impl(
         commitid,
         commit_yaml,
         arguments_list,
+        arguments=None,
```

Comment:
do you know the type of `arguments` or `arguments_list`?

Reply:
yes, I have plans to properly type those as `TypedDict` in the future.